packages/webpack/src/buildtime/policyGenerator.js

```diff
@@ -87,16 +89,18 @@ module.exports = {
        * @param {Iterable<import('webpack').ModuleGraphConnection>} connections
        */
       inspectWebpackModule: (module, connections) => {
+        if (isExcludedUnsafe(module)) return
```

Comment:
here we want to ignore all stuff marked as intended to be ignored, no need to protect ourselves from attempts to self-ignore. We're getting more inputs to inspect than to bundle for reasons I didn't research fully.